study.md

```diff
@@ -43,5 +51,12 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+
+A Hash URL is like `http://thingy.com/#/posts/1/history`. They have `#` in the URL.
+
+It's important not to prefix image data with a `/`. You must add an if statement saying if the environment is a production environment then change the base url to `/project-name` and set the locationtype to `hash`.
+You also need to be aware that you want to copy the compiled data from dist. then push to gh-pages.
```

Comment:
Yes, to host an Ember application on github use `ember build --environment production` to build the project to `dist/`, because it is unable to run a server.